### ISSUE TITLE
make error

### ISSUE THREAD

**Issue by NikosAlexandris** [jez/pandoc-markdown-css-theme#7]
I cloned the repository, installed `pandoc` and `pandoc-sidenote` in a virtual environment via conda/pip. I made sure that `pandoc-sidenote` is in `PATH`. I then follow the instructions: `make clean` and then `make`. Here the error:
```
tools/build.sh "src/paper/index.md" "docs/paper/index.html"
pandoc-sidenote: Error in $: Incompatible API versions: encoded with [1,22] but attempted to decode with [1,20].
CallStack (from HasCallStack):
  error, called at ./Text/Pandoc/JSON.hs:107:64 in pandoc-types-1.20-BxsEriGxqLC6PnOiTnNIPK:Text.Pandoc.JSON
Error running filter pandoc-sidenote:
Filter returned error status 1
make: *** [Makefile:39: docs/paper/index.html] Error 83
```

Any help appreciated. Thank you.

**Comment by NikosAlexandris** [comment]
I see now the table [dependencies table](https://github.com/jez/pandoc-sidenote#dependencies). A bit of a mess with the required dependencies here. I'd need to get some time to resolve the issue.

**Comment by jez** [comment]
You can either upgrade `pandoc` to the latest version, or downgrade `pandoc-sidenote` to `0.20.0` and that should fix the issue.

**Comment by NikosAlexandris** [comment]
My system's pandoc is still
```bash
➜  pandoc --version
pandoc 1.19.2.1
```

So, I created a virtual environment
```
➜  conda create -n pandoc
➜  conda activate pandoc
```
where I installed the latest pandoc
```
➜  pandoc --version
pandoc 2.14.1
```

Note, the [dependencies table](https://github.com/jez/pandoc-sidenote#dependencies) shows version 2.11 for pandoc. Does the table need an update?

So, to be safe, I created another virtual environment, where
```
➜  conda activate pandoc-old
➜  pandoc --version
pandoc 2.11.4
```

Now, I have zero idea about `cabal` and th rest. So, if I specify the version of pandoc-sidenote to install, like
```bash
➜  cabal install pandoc-sidenote-0.22.0
```
it doesn't do because it doesn't see the version inside the virtual environment -- or at least I think this is the problem. It sees the system's version of pandoc and won't continue. The simple call
```bash
✗  cabal install pandoc-sidenote
Resolving dependencies...
Configuring pandoc-sidenote-0.20.0.0...
Building pandoc-sidenote-0.20.0.0...
Installed pandoc-sidenote-0.20.0.0
```
does it. Now, I downgrade pandoc inside the virtual environment to ver. 2.9.
I am linking pandoc-sidenote int the virtual environment's `bin` directory, head on the `make clean` and `make`.

It works, finally. Not that straight... but ok.

Thanks
